sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2535,121 +2557,107 @@ object Radar {
+        callback: RadarLogConversionCallback
     ) {
-        if (!initialized) {
-            callback.onComplete(RadarStatus.ERROR_PUBLISHABLE_KEY)
-
-            return
-        }
+        val nonNullMetadata = metadata ?: JSONObject()
+        nonNullMetadata.put("revenue", revenue);
 
-        trackOnce(location, object : RadarTrackCallback {
-            override fun onComplete(status: RadarStatus, location: Location?, events: Array<RadarEvent>?, user: RadarUser?) {
-                if (status != RadarStatus.SUCCESS || location == null) {
-                    handler.post {
-                        callback.onComplete(status)
-                    }
-
-                    return
-                }
-
-                sendEvent(customType, metadata, location, events, user, callback)
-            }
-        })
+        logConversion(name, nonNullMetadata, callback)
     }
 
     /**
-     * Sends a custom event with a manually provided location.
+     * Logs a conversion with revenue.
      *
      * @see [](https://radar.com/documentation/api#send-a-custom-event)
      *
-     * @param[customType] The name of the event.
-     * @param[location] The location of the event.
-     * @param[metadata] The metadata associated with the event.
+     * @param[name] The name of the conversion.
+     * @param[revenue] The revenue generated by the conversion.
+     * @param[metadata] The metadata associated with the conversion.
      * @param[block] A block callback.
      */
     @JvmStatic
-    fun sendEvent(
-        customType: String,
-        location: Location,
-        metadata: JSONObject?,
-        block: (status: RadarStatus, location: Location?, events: Array<RadarEvent>?, user: RadarUser?) -> Unit
+    fun logConversion(
+        name: String,
+        revenue: Double,
+        metadata: JSONObject? = null,
+        block: (status: RadarStatus, RadarEvent?) -> Unit
     ) {
-        sendEvent(customType, location, metadata, object : RadarSendEventCallback {
+        logConversion(name, revenue, metadata, object : RadarLogConversionCallback {
             override fun onComplete(
                 status: RadarStatus,
-                location: Location?,
-                events: Array<RadarEvent>?,
-                user: RadarUser?
+                event: RadarEvent?
             ) {
-                block(status, location, events, user)
+                block(status, event)
             }
         })
     }
 
     @JvmStatic
-    private fun sendEvent(
-        customType: String,
-        metadata: JSONObject?,
-        location: Location?,
-        events: Array<RadarEvent>?,
-        user: RadarUser?,
-        callback: RadarSendEventCallback
+    internal fun sendLogConversionRequest(
+        name: String,
+        metadata: JSONObject? = null,
+        callback: RadarLogConversionCallback
     ) {
+        if (!initialized) {
+            callback.onComplete(RadarStatus.ERROR_PUBLISHABLE_KEY)
+
+            return
+        }
+
```

Comment:
We can remove this bc `apiClient.sendEvent()` already does this check (and also we're not doing it in the ios sdk)